### PR TITLE
split VH cards

### PR DESCRIPTION
**Pull request by shimashimarin**
Split the VHto2G datacards to W+H, W-H, and ZH. As discussed in MccM [gitlab](https://gitlab.cern.ch/cms-gen/mccm/-/issues/466)

### CODE REVIEW

**Comment by lviliani** [comment]
Thanks! Can you please paste here the size of the gridpacks, also in terms of number of files?

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/WmH012j_5f_NLO_FXFX_M_VToAll/WmH012j_5f_NLO_FXFX_M_VToAll_customizecards.dat
Comment (lviliani) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/WmH012j_5f_NLO_FXFX_M_VToAll/WmH012j_5f_NLO_FXFX_M_VToAll_customizecards.dat:1]: Do we want to also change the width together with the mass?

Comment (shimashimarin) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/WmH012j_5f_NLO_FXFX_M_VToAll/WmH012j_5f_NLO_FXFX_M_VToAll_customizecards.dat:1]: As before, we assume a narrow-width resonance like the Higgs. The width is negligible comapred with the resolution. So, we only need to change the mass

Comment (shimashimarin) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/WmH012j_5f_NLO_FXFX_M_VToAll/WmH012j_5f_NLO_FXFX_M_VToAll_customizecards.dat:1]: Here is the size information of the gridpacks:

| Process    | Gridpack Size (MB) | Untarred Size (GB) | #files  |
| -----------| --------------------- | -------------------- | ------  |
| ZH           |  103                         | 1.6                          | 19680 |
| W+H       |  219                         |  4.7                         | 47466 | 
| W-H        |  219                         |  4.7                         | 47529 |

**Pull request merged**